apis/core/v1beta1/collector_meta.go

```diff
@@ -0,0 +1,10 @@
+package v1beta1
+
+func (c *CollectorSpec) IsEmpty() bool {
+	empty := true
+	if c.LoggingConfig != nil {
+		empty = false
+	}
+
+	return empty
```

Comment:
can probably simplify this to `return c.LoggingConfig != nil`

Reply:
So currently yes, but this function is setting the foundation for future work.  In the near future we will add metrics configuration, and it should only return true if there are neither logging nor metrics config.  So in this case we would have an additional if block e.g.
```go
if c.MetricsConfig != nil {
    empty = false
}
```

Reply:
its definitely a stylistic nitpick but : `c.Metrics == nil && c.LoggingConfig == nil && ...`  works just as well